src/app/views/organization/app-instance-info-modal/app-instance-info-modal.html

```diff
@@ -0,0 +1,139 @@
+<div id="app-instance-info-modal">
+
+  <!-- Modal Header -->
+  <div class="modal-header">
+    <h3 class="modal-title" translate translate-values="{ app_name: vm.appInstance.app_name, organization_name: vm.organization.name}">mnoe_admin_panel.dashboard.organization.app_info.title</h3>
+    <div class="close" type="button" ng-click="vm.closeModal()" >×</div>
+  </div>
+
+  <!-- Modal Body -->
+  <div class="modal-body">
+    <div class="row">
+      <div class="col-xs-6">
+        <div class="label-cell" translate>mnoe_admin_panel.dashboard.organization.app_info.name</div>
+      </div>
+      <div class="col-xs-6">
+        {{ vm.appInstance.app_name }}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-6">
+        <div class="label-cell" translate>mnoe_admin_panel.dashboard.organization.app_info.app_instance_uid</div>
+      </div>
+      <div class="col-xs-6">
+        {{ vm.appInstance.uid }}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-6">
+        <div class="label-cell" translate>mnoe_admin_panel.dashboard.organization.app_info.app_status</div>
+      </div>
+      <!-- TODO: Add appropriate tooltips for app instance statuses. -->
```

Comment:
There are around 10 app instance statuses, and I'm unsure as to what all of them mean. Would be good to pick out 4-5 that we can write informative tooltips about.

Reply:
```ruby
ALL_STATUSES = %w(starting restarting running stopping stopped terminating terminated updating upgrading downgrading)
```
Most of them are used for hosted app (cubes) which are being deprecated.
You can just focus on these, which are the only one relevant for most of the connectors:
* running
* terminated


They're self-explanatory so not sure we'd need a tooltip?

Reply:
@ouranos Great.